### ISSUE TITLE
Add RGB(A)->HSV(A) conversion

### ISSUE THREAD

**Issue by strarsis** [voxpelli/sass-color-helpers#7]
It would be great if also conversion from RGB(A) to HSV(A) is added.

**Comment by voxpelli** [comment]
@strarsis Makes sense to support it, what do you think about the code I added in a PR?

**Comment by voxpelli** [comment]
@strarsis No feedback?

**Comment by strarsis** [comment]
Thanks for the PR, the different color system should result in different alpha mixing.
(Originally I wondered how to generally convert RGB to HSV. - 
Apparently this was is possible by just passing a RGB value to ch-hsl-to-hsv(...).)
